plugins/search-backend-module-api-docs/api-report.md

```diff
@@ -0,0 +1,89 @@
+## API Report File for "@backstage/plugin-search-backend-module-api-docs"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+/// <reference types="node" />
+
+import { CatalogApi } from '@backstage/catalog-client';
+import { Config } from '@backstage/config';
+import { DocumentCollatorFactory } from '@backstage/plugin-search-common';
+import { IndexableDocument } from '@backstage/plugin-search-common';
+import { Logger } from 'winston';
+import { OpenAPI } from 'openapi-types';
+import { OpenAPIV2 } from 'openapi-types';
+import { OpenAPIV3 } from 'openapi-types';
+import { PluginEndpointDiscovery } from '@backstage/backend-common';
+import { Readable } from 'stream';
+import { TokenManager } from '@backstage/backend-common';
+
+// @public (undocumented)
+export interface APIDocument extends IndexableDocument {
+  kind: string;
+  lifecycle: string;
```

Comment:
Right now we're returning the following by default:

```
title: entity.metadata.name,
location: `/catalog/default/api/${entity.metadata.name}/definition/`,
text: specParser.getSpecText(entity.spec?.definition as string),
kind: entity.kind,
lifecycle: (entity.spec?.lifecycle as string) || '',
```

I could maybe see folks wanting to see extra fields, but I'm maybe that should be split out into it's own PR? Also, curious if we actually need. `APIDefinitionDocument`, since it pretty much overlaps with `CatalogEntityDocument`. Maybe if folks would want to see something specific to an APIDefinition like 'provided by` ?

Reply:
At the moment, nothing comes to mind. It'd be great if the @backstage/maintainers would give their opinion. I'd also suggest sharing on discord so we can get ideas from others 🙂 .